dashboard/app/views/regional_partners/_form.html.haml

```diff
@@ -25,6 +25,12 @@
   .field
     = f.label nil, "Apps Close Date (YYYY-MM-DD)"
     = f.text_field :apps_close_date_teacher
+  .field
+    = f.label nil, "Underrepresented Group Guardrail Percent"
+    = f.number_field :urg_guardrail_percent, :in => 0..100
+  .field
+    = f.label nil, "Free and Reduced Lunch Guardrail Percent"
+    = f.number_field :frl_guardrail_percent, :in => 0..100
```

Comment:
Love the specificity of the help tips on the frontend! Thanks for making that demo video